### PR TITLE
fix bug for https://github.com/Adldap2/Adldap2-Laravel/issues/21

### PR DESCRIPTION
**Pull request by minkbear**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adldap2/adldap2-laravel",
+    "name": "minkbear/adldap2-laravel",
```
Comment (strebl) [inline composer.json]: Why did you modify the package name?

Comment (minkbear) [inline composer.json]: sorry I do not know when my PR will be merge and I have a deadline to use my modified package by using composer(https://packagist.org/) to be sure that I can release for my client.

I welcome to send new PR. what do you think?

Comment (strebl) [inline composer.json]: Don't do that. You should use composer's repositories for that.

https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository

**Comment by stevebauman** [comment]
Sorry, this PR doesn't resolve the actual issue at hand. A user instance must be returned.

**Pull request closed without merging**